### DISCUSSION TITLE
Snapshot testing and random name

### DISCUSSION THREAD

**Discussion by yhabib** [ariakit/ariakit#789] [Questions] [answered]
First thanks for this library 🙌 

I wanted to ask what is the recommended approach for snapshot testing when using for example the Radio button that uses random values for name. Is it somehow possible to provide this value to `useRadioState` to use a custom value for `name` ?

**Comment by diegohaz** [comment, marked as answer]
You can pass `baseId` to `useRadioState` (it works for all modules that have random ids or names).
```jsx
useRadioState({ baseId: "my-base-id" });
```
